apps/builder/app/builder/features/props-panel/control.tsx

```diff
@@ -201,34 +188,15 @@ export function Control({
   onChangePropValue,
   setCssProperty,
 }: ControlProps) {
-  const meta = getComponentMetaProps(component);
+  const meta = getComponentMeta(component)?.props;
 
   const argType = meta?.[userProp.name];
 
   // argType can be undefined in case of new property created
   const defaultValue = argType?.defaultValue ?? "";
-  const type = argType?.type ?? "text";
-
-  if (type == null) {
-    warnOnce(
-      true,
-      `No control type for prop "${userProp.name}" component "${component}" found`
-    );
-    return <NotImplemented />;
-  }
-
-  if (typeof type !== "string") {
-    warnOnce(
-      true,
-      `Control type "${typeof type}" for prop "${
-        userProp.name
-      }" component "${component}" is not a string`
-    );
-
-    return <NotImplemented />;
-  }
+  const controlType = argType?.controlType ?? "text";
 
-  if (component === "Image" && userProp.name === "src") {
+  if (controlType === "imageUrl") {
```

Comment:
https://storybook.js.org/docs/react/essentials/controls#annotation

See {control: {file, accept}} ...

Reply:
Do you mean I should rename `imageUrl` to `file` and add support of `accept` option?

Reply:
yes

Reply:
Will do this when I'll get to implementing the `file` control https://github.com/webstudio-is/webstudio-builder/issues/1011